src/player.js

```diff
@@ -356,6 +367,11 @@ export default class Player extends FakeEventTarget {
     this._firstPlay = true;
     this._fullscreen = false;
     this._firstPlayInCurrentSession = true;
+    this._repositionCuesTimeout = false;
+    this._loadingMedia = false;
```

Comment:
hey, whats the the difference between _loading & _loadingMedia?

Reply:
Those flags are not part of this change but in general:
`loading` means that `load()` has been called on the player but the load promise isn't resolved yet, i.e while loading the engine. This is to prevent multiple load calls while a previous call is already processing. 
`loadingMedia` flag is a workaround that we should remove. Its means that `loadMedia()` has been called on the kaltura player but the provider response isn't resolved yet.